jit-compiler/src/main/java/org/aya/compiler/serializers/MatchySerializer.java

```diff
@@ -72,9 +72,10 @@ private void buildInvoke(
 
     var matching = unit.clauses().map(clause ->
       new PatternSerializer.Matching(clause.bindCount(), clause.patterns(),
-        (ps, cb, bindCount) -> {
-          var resultSeq = AbstractExprializer.fromSeq(cb, Constants.CD_Term, ps.result.ref(), bindCount);
+        (ps, cb, _) -> {
+          var resultSeq = ps.result.map(LocalVariable::ref);
```

Comment:
I am not sure if all `result` are available (not null).

Reply:
Sorry I don't think I follow. I think this should be fine as long as `ps.result` isn't null? `result` should be a sequence of non-null `LocalVariable` populated in `PatternSerializer#serialize`.

Reply:
All `LocalVariable` is always not null, but not the value they represent at runtime. The value of some `result` may be null and cannot be used for `serializeTermUnderTele`

Reply:
I mean we should use the 3rd parameter.

Reply:
Ah I see.